Azure/sonic-buildimage
# Add  snh60b0-640 and snh60a0-320fv2  devices

ASIC Vendor: Broadcom
Swich ASIC:
  1.Tomhawk2 chip: for snh60b0-640f device
  2.Tomhawk chip: for snh60a0-320fv2 device

**- What I did**
  1. Add new platform snh60b0-640f
  2. Add new platform snh60a0-320fv2

**- How I did it**
  Provide device and plarform related files.
**- How to verify it**
	How to verify it
	psuutil status
	sfputil show presence
	sfputil show eeprom
	decode-syseeprom
	sensors
	fan policy check
**- Description for the changelog**

**- A picture of a cute animal (not mandatory but encouraged)**
